content/docs/user-guide/packaging/project-export/project-export-android.md

```diff
@@ -0,0 +1,117 @@
+---
+linkTitle: Project Export for Android
+title: Project Export for Android
+description: Learn how to use the Project Export CLI to automate preparing your project for release on Android.
+toc: true
+weight: 410
+---
+This guide covers how to use project export tooling to build and deploy games on Android. We will use the Android export script to generate a gradle project and prepare assets and then build the project. The end result of export should be an APK file that can be installed on your Android device, along with an additional step to auto-install on a connected phone.
+
+{{< note >}}
+To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](/docs/user-guide/packaging/project-export/project-export-cli)
+{{< /note >}}
+{{< note >}}
+To learn more about how to manually export an Android project, please consult the page [Generating an Android Project](https://development--o3deorg.netlify.app/docs/user-guide/platforms/android/generating_android_project_windows/#prerequisites).
+{{< /note >}}
+
+## Prerequisites
+1. Make sure that the [Project Export CLI Tool page](/docs/user-guide/packaging/project-export/project-export-cli) prerequisites are satisfied.
+2. Make sure you have your project created and registered with your engine. Check that you can successfully build and open your project with the Project Manager.
+3. Ensure all pre-requisites for working with Android in O3DE are satsified. You can learn about software dependencies [here](https://development--o3deorg.netlify.app/docs/user-guide/platforms/android/#prerequisite-software-and-packages) and project setup pre-requisites [here](https://development--o3deorg.netlify.app/docs/user-guide/platforms/android/generating_android_project_windows/#prerequisites). For the Android SDK, make sure to record where the root folder path is located on your hard drive, as that will be needed for setting up configuration.
+4. Ensure that you have a proper keystore file configured for signing. You can learn more [here](https://development--o3deorg.netlify.app/docs/user-guide/platforms/android/#apk-signing). Here is an example keystore file you can create to start with (you will need the JDK with `keytool`):
+```
+export KEYSTORE_FILE_PATH=/path/to/android-key.keystore
+export STOREPASS=o3depass
+export ALIAS=o3dekey
+export KEYPASS=o3depass
+export KEYSIZE=2048
+export VALIDITY=10000
+export DNAME="cn=o3de-android-project, ou=o3de, o=LF, c=US"
+keytool -genkey -keystore $KEYSTORE_FILE_PATH -storepass $STOREPASS  -alias $ALIAS -keypass $KEYPASS -keyalg RSA -keysize $KEYSIZE -validity $VALIDITY -dname $DNAME
+```
+5. Make sure the o3de engine bootstrap registry is enabled for android. To do so, go to your O3DE engine installation and edit the file at `$O3DE_ENGINE_PATH/Registry/bootstrap.setreg` so that the `assets` field uses `android`. Like so:
```

Comment:
```suggestion
5. Make sure the O3DE bootstrap registry is enabled for Android. To do so, go to your O3DE installation and edit the file at `$O3DE_ENGINE_PATH/Registry/bootstrap.setreg` so that the `assets` field uses `android`. Like so:
```